packages/eslint-config-react-native/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "eslint-config-react-native",
+  "description": "STRV's ESLint config for React Native projects",
+  "version": "1.0.0",
+  "author": "André Nanninga <andre@nannin.ga>",
+  "bugs": {
+    "url": "https://github.com/strvcom/code-quality-tools/issues"
+  },
+  "keywords": [
+    "config",
+    "eslint",
+    "react-native",
+    "strv"
+  ],
+  "license": "BSD-3-Clause",
+  "main": "index.js",
+  "directories": {
+    "lib": "lib",
+    "test": "__tests__"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/strvcom/code-quality-tools.git"
+  },
+  "peerDependencies": {
+    "eslint": "^5.3.0"
+  },
+  "publishConfig": {
+    "access": "public"
+  }
+}
```

Comment:
Please add `engines.node >=6` stanza to declare minimum Node.js runtime supported. See other _packge.json_ files for pointers. 🙏